horovod/common/operations.cc

```diff
@@ -854,7 +870,63 @@ void CoordinateCacheAndState(CacheCoordinator& cache_coordinator,
 //      otherwise we may end up dispatching many blocked threads and never make
 //      progress if we have a thread pool limit.
 bool RunLoopOnce(HorovodGlobalState& state, MPIContext& ctx, bool is_coordinator);
+
+#if HAVE_MLSL
```

Comment:
Can we move this into `MLSLContext`?  One of our goals with `operations.cc` is to minimize the amount of "framework specific" code in here.  Ideally, we want to isolate framework code within specific places like `mlsl_context.h` or `mlsl_operations.h`.

Reply:
Moved to MLSLContext